components/pipeline-service/development/kustomization.yaml

```diff
@@ -8,7 +8,55 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
+  - github.com/minio/operator?ref=v5.0.15
```

Comment:
It's mentioned in the commit message that the newer version requires k8s 1.28 (OCP 4.15) and that's already the case for the development (clusters used in CI) and the Konflux staging environment. Users also can't use earlier versions because the currently used version of OSP requires k8s 1.28. SO if there isn't other reason, you can use 6.0.3.
The rest is LGTM.